### ISSUE TITLE
Video Stream Merger

### ISSUE THREAD

**Issue by lmangani** [meething/meething#78]
Potential candidate for a (powerful-browser-based) MCU for videowalling over a single stream. Just exploring and leaving note here for others to check out: 

https://github.com/t-mullen/video-stream-merger

**Comment by QVDev** [comment]
Nice and actually used it and was able to record stream from multiple users. But using canvas is to heavy though. But we should.look.into.porting this to offscreen canvas in webworker that will probably do the trick. But then you might be quicker writting from scratch

**Comment by lmangani** [comment]
Indeed, but If we could even just merge 4 users into a stream it would be a successful experiment.

**Comment by QVDev** [comment]
Ah that is doable though non worries. I trough my code away but can hack.something together. Do we want one mediastream right as output and we sent that accross?

**Comment by QVDev** [comment]
https://slides.com/twoxaa/woah-is-that-offscreencanvas/fullscreen#/0/12
